activerecord/lib/active_record/railtie.rb

```diff
@@ -171,23 +171,35 @@ class Railtie < Rails::Railtie # :nodoc:
     end
 
     initializer "active_record.define_attribute_methods" do |app|
+      # For resiliency, it is critical that a Rails application should be
+      # able to boot without depending on the database (or any other service)
+      # being responsive.
+      #
+      # Otherwise a bad deploy adding a lot of load on the database may require to
+      # entirely shutdown the application so the database can recover before a fixed
+      # version can be deployed again.
+      #
+      # This is why this initializer tries hard not to query the database, and if it
+      # does, it makes sure to any possible database error.
```

Comment:
Minor typo here @casperisfine 

Is it supposed to be?
```suggestion
      # does, it makes sure to rescue any possible database error.
```

Reply:
Thanks, fixed in 47d0f4ec0f7fd6f3919d37dc158ae25e3cedcaab